vscode/microsoft-kiota/src/extension.ts

```diff
@@ -225,7 +225,7 @@ export async function activate(
      clientOrPluginKey = clientKey;
      clientOrPluginObject = clientObject;
      workspaceGenerationType = generationType;
-     await loadEditPaths(clientObject, openApiTreeProvider);
+     await loadEditPaths(clientOrPluginKey, clientObject, openApiTreeProvider);
```

Comment:
This is only called when we regenerate a client/plugin. Should we also update the value in after the first generation is perfomed?

Reply:
Yeah, I wanted to create a separate PR for this work